packages/yoastseo/src/scoring/assessments/KEYPHRASE MATCHING.md

```diff
@@ -0,0 +1,50 @@
+
+##### Premium
+All Premium users can also add **synonyms** of their keyphrase. Some of the assessments will then not only consider the keyphrase, but also its synonyms.
+
+#### Group 2: Languages with function word support
+Here is the [list of languages with function word support](https://github.com/Yoast/wordpress-seo/tree/trunk/packages/yoastseo/README.md#seo-analysis).
+
+##### Free
+In addition to per-word matching, we also filter out **function words** from the keyphrase during keyphrase matching. For example, if the keyphrase is "how to decorate a room", we filter out the words 'how', 'to', and 'a', and only leave out the meaningful words 'decorate' and 'room'. This means that if we look, for example, if the keyphrase has been used in the introduction of the text, we are only looking for the words 'decorate' and 'room'.
+
+##### Premium
+In addition to the above, Premium users can add **synonyms** of their keyphrase.
+
+#### Group 3: Languages with word form support
+Here is the list of [languages with word form support](https://github.com/Yoast/wordpress-seo/tree/trunk/packages/yoastseo/MORPHOLOGY.md).
+
+##### Free
+Free users with languages that have word form support have access to the same functionalities as Free users of group 2 (filtering out **function words** and **per-word matching**)
+
+##### Premium
+Premium users have access to **full word form support**. This means that in addition to all other functionalities mentioned above, **all possible forms of their keyphrase and all possible forms of their synonyms** are recognized. For example, if the keyphrase is "room decorating ideas" and the synonym is "apartment decorating ideas" all the possible forms of each word are recognized: 'apartments', 'rooms', 'room's', 'decorate', 'decorates', 'decorated', 'idea', etc. These are then taken into account during keyphrase matching, after possible function words have been filtered out.
+
+#### Summary
+|    	      | All languages	     | Languages with function word support | Languages with word form support 	                          |
+|------------	|------------------	|---------------------	|-------------------------------------------------------------|
+| Free        | Per-word matching    | Per-word matching, filtering out function words | Per-word matching, filtering out function words             |
+| Premium	  | The above + synonyms | The above + synonyms	      | The above + synonyms + word forms of keyphrase and synonyms |
+
+#### Important notes
+* Per-word matching, filtering out of function words and morphological forms of keyphrase and synonyms **can be suppressed** by using double quotations marks around the keyphrase or around a synonym. E.g., `"SEO analysis"`.
```

Comment:
Ah this is the explanation about the exact matching, but maybe we can make this clearer that this is what we meant by "exact matching"? Also somehow refer this explanation to my comment above about "exact matching"